homeassistant/components/whirlpool/const.py

```diff
@@ -1,3 +1,11 @@
 """Constants for the Whirlpool Sixth Sense integration."""
 
+from whirlpool.backendselector import Region
+
 DOMAIN = "whirlpool"
+CONF_ALLOWED_REGIONS = ["EU", "US"]
+
+CONF_REGIONS_MAP = {
+    "EU": Region.EU,
+    "US": Region.US,
+}
```

Comment:
You could directly use the `list(CONF_REGIONS_MAP)` or `in CONF_REGIONS_MAP` depending on the use to avoid duplicating the names if you wish.

Reply:
I assume this can be marked as resolved (as-in, not to be changed), @abmantis?

Reply:
Thanks for reminding @rytilahti ! I just removed the redundant list.